com.unity.netcode.gameobjects/Runtime/Messaging/MessageQueue/MessageQueueHistoryFrame.cs

```diff
@@ -6,7 +6,7 @@ namespace Unity.Netcode
     /// <summary>
     /// Used by the MessageQueueContainer to hold queued messages
     /// </summary>
-    public class MessageQueueHistoryFrame
+    internal class MessageQueueHistoryFrame
```

Comment:
Bad merge with develop? Doesn't seem like it belongs...

Reply:
intentional but will move under a separate PR.